webpack/assets/javascripts/react_app/components/Layout/components/TaxonomySwitcher/TaxonomyDropdown.scss

```diff
@@ -0,0 +1,5 @@
+.pf-c-context-selector__menu-list-item a.organization_menuitem,
+.pf-c-context-selector__menu-list-item a.location_menuitem {
```

Comment:
Is there a way to avoid using the PF class-names and assign our own class-names instead?
So if PF will change there class-names as they did, less chance for breaking changes.

Reply:
Changed to not include PF class-names